### DISCUSSION TITLE
Local or cloud?

### DISCUSSION THREAD

**Discussion by VACInc** [humbertogontijo/homeassistant-roborock#170] [General]
Once authenticated, does the integration require cloud connection?

**Comment by JurajNyiri** [comment]
Yes, this integration uses only cloud.

**Comment by StaleLoafOfBread** [comment]
This uses the cloud. There has been some discussion on the VLAN thread. I personally think its possible to make it work local but this is just far enough out of my wheelhouse that I don't have the time to dig into it much.  https://github.com/humbertogontijo/homeassistant-roborock/discussions/96#discussioncomment-4595170
